Getting started guide (#15)
neo4j/cypher-builder: docs/modules/ROOT/pages/getting-started/querying.adoc

= Querying

> In this guide, you'll create a simple Cypher query to match and return Movies using the Cypher Builder:

Reviewer: ```suggestion
This tutorial shows how to create a simple Cypher query to match and return Movies using the Cypher Builder:
```
